src/Type/JsonDocumentType.php

```diff
@@ -109,6 +109,10 @@ public function convertToDatabaseValue($value, AbstractPlatform $platform)
      */
     public function convertToPHPValue($value, AbstractPlatform $platform)
     {
+        if (null === $value || '' === $value) {
```

Comment:
It should return an empty string `''` if the column contains an empty string and `null` if it contains `null`.